src/components/Canvas.tsx

```diff
@@ -13,7 +13,11 @@ type lineType = {
   points: number[];
 };
 
-const Canvas = () => {
+interface Props {
+  stageRef: React.RefObject<Konva.Stage>;
+}
+
+const Canvas = ({ stageRef }: Props) => {
```

Comment:
関数コンポーネントの型定義は僕はこうするんですが、こちらの方法はどうですか？

```suggestion
const Canvas: React.VFC<Props> = ({ stageRef }) => {
```

Reply:
propsのときはinterfaceを使うことが多いですが、typeも変わらないのであとはルールの決めの問題ですかね。笑
今回はtypeを使うことにしますか！